runtime/vm/jvminit.c

```diff
@@ -3556,6 +3556,18 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 	}
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	{
+		IDATA enableCRIU = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXENABLECRIU, NULL);
+		IDATA disableCRIU = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXDISABLECRIU, NULL);
+		if (enableCRIU > disableCRIU) {
+			vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_CRIU_SUPPORT;
+		} else if (enableCRIU < disableCRIU) {
```

Comment:
This isn't really necessary since the flag isn't set by default. I could have an effect in the future if J9_EXTENDED_RUNTIME2_ENABLE_CRIU_SUPPORT is set by default later.